src/test/regress/sql/citus_local_tables_mx.sql

```diff
@@ -381,5 +381,160 @@ SELECT run_command_on_workers(
 $$
 SELECT count(*) FROM pg_catalog.pg_tables WHERE tablename='citus_local_table_4'
 $$);
+
+-- test adding foreign table to metadata with the guc
+SET citus.use_citus_managed_tables TO ON;
```

Comment:
this test file is already too large. How hard would it be to move these to another file? If simple, lets do that, its much preferable. 

Otherwise, would be very useful to add some comments on each part of the that on what we are actually testing.